node/coinstacks/gnosis/pulumi/Pulumi.yaml

```
@@ -0,0 +1,3 @@
+name: gnosis
+runtime: nodejs
+description: coin stack indexer, ingestion and interface
```

some eth things are a little out of date, been just using `coin stack`
```suggestion
description: coin stack
```